test/models/PenaltyEntrySchema.test.ts

```diff
@@ -131,6 +131,42 @@ describe('Penalty Details Schema Validation', () => {
                 const result = validator.validate(createModelWithPenaltyReference('z12345678'));
                 expect(result).to.deep.equal({errors: []});
             });
+
+            it('should accept legacy penalty references', () => {
+                const result = validator.validate(createModelWithPenaltyReference('PEN1A/SC123123'));
+                expect(result).to.deep.equal({errors: []});
+            });
+
+            it('should accept lower case legacy penalty references', () => {
+                const result = validator.validate(createModelWithPenaltyReference('pen1A/sc123123'));
+                expect(result).to.deep.equal({errors: []});
+            });
+
+            it('should accept legacy penalty references with leading zeros', () => {
+                const result = validator.validate(createModelWithPenaltyReference('pen1A/sc123'));
```

Comment:
are you missing the leading zeros?

Reply:
SC123 is a valid company number, it will be saved on our system as SC000123

Reply:
ill change the name